plugins/generator-1.16.5/forge-1.16.5/mappings/villagerprofessions.yaml

```diff
@@ -11,4 +11,5 @@ LIBRARIAN: VillagerProfession.LIBRARIAN
 MASON: VillagerProfession.MASON
 SHEPHERD: VillagerProfession.SHEPHERD
 TOOLSMITH: VillagerProfession.TOOLSMITH
+WANDERING_TRADER: WanderingTrader
```

Comment:
I'd put this **special case** at the end of the list.

Reply:
You're right, it's done